zenip-42202.md

```diff
@@ -0,0 +1,119 @@
+
+# ZenIP-42202
+
+    ZenIP: 42202
+    Title: Non-ceasable Sidechains
+    Owner: Paolo Tagliaferri, <paolotagliaferri@horizenlabs.io>; cronic, <cronic@horizenlabs.io>
+    Discussions-To: Paolo Tagliaferri, <paolotagliaferri@horizenlabs.io>
+    Comments-URI: https://horizen.global/invite/discord
+    Status: Draft
+    Type: Consensus
+    Created: 2022-08-01
+    License: MIT
+
+## Table of Contents
+
+<!--ts-->
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Specification](#specification)
+  - [Constraints](#constraints)
+  - [Maturity](#maturity)
+
+
+## Abstract
+
+This document describes the introduction of a new type of sidechains that “never cease” from processing transactions. The version for this type of sidechains is “v2”.
```

Comment:
"This document introduces a new type of sidechain that never ceases from processing transactions."

Reply:
Thank you @zain-cheng, I've applied your suggestion!